omigami/plotting/comparison_plotter.py

```diff
@@ -0,0 +1,79 @@
+from typing import List, Optional, Any, Tuple, Union
+
+import pandas as pd
+
+import matplotlib.pyplot as plt
+from omigami.utilities.gnps_helper import GnpsHelper
+from omigami.utilities.spectrum_df_helper import SpectrumDataFrameHelper
+
+
+class SpectraComparisonPlotter:
+    def __init__(self):
+        pass
+
+    def mirror_plot(
+        spectrum_1: Union[pd.DataFrame, str],
+        spectrum_2: Union[pd.DataFrame, str],
+        labels: Optional[List[str]] = None,
+        display_limits: Optional[Tuple[int, int]] = (0, 5000),
+    ) -> Any:
+        """
+        Creates a mirror plot with spectrum 1 on top (blue), and spectrum 2 on the bottom (red).
+
+        Requires:
+        spectrum1 : A dataframe version of the spectrum information for top spectrum
+        spectrum2 : A dataframe version of the spectrum information for bottom spectrum
+
+        Optional:
+        match_id (int): If the dataframes are a
+        labels (list): A list containing the names of the spectra.
+        Display_Limits (list): left and right boundaries for displaying the spectrum.
+
+        Returns:
+        fig (plot): The mirror plot.
+        """
+
+        if not labels:
+            labels = ["Spectrum 1", "Spectrum 2"]
+
+        gnps_helper = GnpsHelper()
+        sdf_helper = SpectrumDataFrameHelper()
+
+        spectra = [spectrum_1, spectrum_2]
+        spectra_df_list = []
+        for spectrum in spectra:
+            if type(spectrum) == str:
+                spectrum_df = gnps_helper.fetch_metadata(spectrum)
+            elif type(spectrum) == pd.DataFrame:
+                if ["m/z", "Intensities"] not in spectrum.columns.values:
+                    raise ValueError(
+                        "Spectra arguments of type DataFrame must contain 'Intensity' and 'm/z' columns."
```

Comment:
Is it `Intensities` or `Intensity` I would say lets be specific and consistent here

Reply:
Thank you, you are right, I went now for "Intensity". This now agrees with the code in SpectrumDataFrameHelper